src/global/utils.ts

```diff
@@ -0,0 +1,36 @@
+import $rdf, { Fetcher, IndexedFormula, NamedNode, sym } from "rdflib"
+import namespace from "solid-namespace"
+
+const ns = namespace($rdf)
+
+export function getName (store: IndexedFormula, user: NamedNode): string {
+  return (store.anyValue as any)(user, ns.vcard("fn"), null, user.doc()) ||
+    (store.anyValue as any)(user, ns.foaf("name"), null, user.doc()) ||
+    user.uri
+}
+
+export function getPod (): NamedNode {
+  // TODO: This is given that mashlib runs on NSS - might need to change when we want it to run on other Pod servers
```

Comment:
Not that we have any process for actually dealing with them, but given that Tim has been using `@@` to signify TODOs, apparently, maybe we should use that too? (Or perhaps just use `@@ TODO`, so it's clear to all of us? :P)

Reply:
// @@ TODO works for my IDE as well (WebStorm), so lets go for this approach in the code we introduce and change when appropriate.